portfolio/src/main/webapp/index.html

```diff
@@ -34,25 +34,26 @@ <h2>A few bullet points about me</h2>
           <ul> 
             <li>Studying Computer Science and Linguistics at Brown University</li>  
             <li>Currently: Building Web App at Google (STEP internship)</li>
-            <li>Hobbies: playing the Clarinet, reading novels, (trying to) talk in Spanish</li> 
             <li>Find me on<a href="https://linkedin.com/in/hollyyuqizheng" target="_blank"> LinkedIn</a> 
                 and<a href="https://github.com/hollyyuqizheng" target="_blank"> GitHub</a></li> 
           </ul> 
+          <h2 onClick="getQuotes()">(Click me) Some of my favorite quotes from books</h2>
+          <ul id="quoteWrapper"></ul>
         </div> 
       
         <div id="projects" class="navbarSection">
           <h2>What I have been working on</h2> 
           <h3>(Click on each project to see more details!)</h3> 
           <div id="projectWrapper">
-            <div id="projectNlp" onClick="addProjectDescription('Nlp')"> 
+            <div id="projectNLP" onClick="addProjectDescription('NLP')"> 
```

Comment:
I would recommend that we adhere to the camel case, even with acronyms.

ex:

projectNlp
projectDescriptionNlp
...

Reply:
Done. Another change I made was changing the parameter passed into the addProjectDescription method : because I need the parameter to match the style of each element's naming, now I pass in "Nlp" instead of "NLP" as the parameter so that "projectDescriptionNlp" can be found. I think passing in "NLP" in all-caps and then trying to re-format this string into "Nlp" in the JavaScript file will be unnecessarily complicated.